src/shims/foreign_items.rs

```diff
@@ -604,6 +604,15 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 this.write_scalar(Scalar::from_u64(n.to_bits()), dest)?;
             }
 
+            "_ldexp" | "ldexp" => {
```

Comment:
Why the `_`?

Reply:
Apparently the windows primitive is called `_ldexp`: https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/floating-point-primitives?view=vs-2019

Reply:
I see, makes sense. Please add a comment (like with `_hypot`).